### PR TITLE
Update the using-a-git-provider-access-token.adoc to omit scm-userid

### PR DESCRIPTION
**Pull request by vinokurig**
## What does this pull request change?
Align the using-a-git-provider-access-token.adoc with the PAT flow simplification: `che.eclipse.org/scm-userid` annotation is omitted.

**Do NOT MERGE** until https://github.com/eclipse-che/che-server/pull/496 is merged.

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/CRW-4182

## Specify the version of the product this pull request applies to
7.64
## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Comment by max-cx** [comment]
I'll review this PR, planned for today.
ACK on postponing the merge.

File: modules/end-user-guide/pages/using-a-git-provider-access-token.adoc [hunk 1]
```diff
@@ -87,7 +80,7 @@ type: Opaque
 <1> Your {prod-short} user ID.
 <2> The Git provider name: `github` or `gitlab` or `bitbucket-server` or `azure-devops`.
 <3> The Git provider URL.
-<4> Your Git provider user ID wrapped in `' '` or `" "`.
+<4> The Git provider username, user email in case you are using `azure-devops`.
```
Comment (max-cx) [inline modules/end-user-guide/pages/using-a-git-provider-access-token.adoc]:
```suggestion
<4> The Git provider username or user's email address for `azure-devops`.
<4> The Git provider username. For `azure-devops`, the user's email address.
```

File: modules/end-user-guide/pages/using-a-git-provider-access-token.adoc [hunk 2]
```diff
@@ -87,7 +80,8 @@ type: Opaque
 <1> Your {prod-short} user ID.
 <2> The Git provider name: `github` or `gitlab` or `bitbucket-server` or `azure-devops`.
 <3> The Git provider URL.
-<4> Your Git provider user ID wrapped in `' '` or `" "`.
+<4> The Git provider username or user's email address for `azure-devops`.
+<4> The Git provider username. For `azure-devops`, the user's email address.
```
Comment (vinokurig) [inline modules/end-user-guide/pages/using-a-git-provider-access-token.adoc:83]:
```suggestion
```

File: modules/end-user-guide/pages/using-a-git-provider-access-token.adoc [hunk 3]
```diff
@@ -87,7 +80,7 @@ type: Opaque
 <1> Your {prod-short} user ID.
 <2> The Git provider name: `github` or `gitlab` or `bitbucket-server` or `azure-devops`.
 <3> The Git provider URL.
-<4> Your Git provider user ID wrapped in `' '` or `" "`.
+<4> The Git provider username or user's email address for `azure-devops`.
```
Comment (max-cx) [inline modules/end-user-guide/pages/using-a-git-provider-access-token.adoc]:
```suggestion
<4> The Git provider username. For `azure-devops`, the user's email address.
```

**Review by max-cx — APPROVED** [review]

**Pull request merged**